src/datasources/asset/AssetDataSource.test.ts

```diff
@@ -207,18 +210,628 @@ const assetsResponseMock: AssetsResponse =
   "totalCount": 4
 }
 
-
 const assetMetadataQueryMock: AssetMetadataQuery = {
+  type: AssetQueryType.Metadata,
   workspace: '',
   refId: '',
   minionIds: ['123']
 }
 
 const buildMetadataQuery = getQueryBuilder<AssetMetadataQuery>()({
+  type: AssetQueryType.Metadata,
   workspace: '',
   minionIds: [],
 });
 
+const assetUtilizationHistoryQueryMock = {
+  "assetUtilizations": [
+    {
+      "utilizationIdentifier": "cf459917-2873-4702-9636-42e4ed1a22f2",
+      "assetIdentifier": "cc87e897-79e2-4cbf-9655-1b145d78a306",
+      "minionId": "System_Product_Name--SN-System_Serial_Number--MAC-40-B0-76-44-7F-26",
+      "category": "Configuration",
+      "taskName": "NI SystemLink Notifications",
+      "userName": "ServEng",
+      "startTimestamp": "2023-12-22T05:42:18.462Z",
+      "endTimestamp": "2023-12-22T06:09:29.686Z",
+      "heartbeatTimestamp": "2023-12-22T06:07:18.511Z"
+    },
+    {
+      "utilizationIdentifier": "1de94e14-c8fa-4ef6-8278-fdb8f0798076",
+      "assetIdentifier": "cc87e897-79e2-4cbf-9655-1b145d78a306",
+      "minionId": "System_Product_Name--SN-System_Serial_Number--MAC-40-B0-76-44-7F-26",
+      "category": "Configuration",
+      "taskName": "NI SystemLink Notifications",
+      "userName": "ServEng",
+      "startTimestamp": "2023-12-22T05:26:39.586Z",
+      "endTimestamp": "2023-12-22T05:31:07.594Z"
+    },
```

Comment:
At a glance, it looks like the only thing changing between the entries of this array are the timestamps. Maybe we could build it programmatically instead of having to inline this giant object.